### PR TITLE
TextSize from https://github.com/FormidableLabs/victory-chart/pull/357

### PR DESCRIPTION
**Pull request by nightwolfdu**
TextSize from https://github.com/FormidableLabs/victory-chart/pull/357 
Add test for width with angle and line-height.

### CODE REVIEW

**Comment by nightwolfdu** [comment]
Сollapse commit's into 1 after review.
Do i need commit /dist/.\* changes?

File: src/victory-util/textsize.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+    result = (fontSize ? value * fontSize : value * coefficients.defaultFontSize)
+      * relativeMeasurementUnitsCoef[attribute];
+  } else {
+    result = value;
+  }
+  return result;
+};
+
+const prepareParams = (inputStyle) => {
+  const style = merge({}, defaultStyle, inputStyle);
```
Comment (boygirl) [inline src/victory-util/textsize.js]: Does this need to be recursive `merge`? Prefer `defaults` where possible.

**Comment by boygirl** [comment]
@nightwolfdu this looks great! Thanks for extracting this work into victory-core. I have only one small comment about `merge` vs. `defaults` but other than than, ready to merge. And no need to commit dist/. That's only for release.

**Comment by nightwolfdu** [comment]
Replace _merge_  with _defaults_, **fetch & rebase** on master, fix conflict. 
Ready to merge! =)

**Comment by boygirl** [comment]
🎉 Thanks so much for this feature!

**Pull request merged**